src/pipeline/errs.js

```diff
@@ -0,0 +1,18 @@
+export const originServerConnectionTerminated = 'originServerConnectionTerminated';
+export const cantResolveUrl                   = 'cantResolveUrl';
+export const originServerRequestTimeout       = 'originServerRequestTimeout';
+
+export function getMsg (err, url) {
+    switch (err) {
+        case originServerConnectionTerminated:
+            return `Failed to perform a request for the "${url}" resource because origin server connection was unexpectedly terminated.`;
```

Comment:
Failed to perform a request for the resource at "${url}" because connection to an origin server was unexpectedly terminated.